hmpps-sqs-spring-boot-autoconfigure/src/main/kotlin/uk/gov/justice/hmpps/sqs/HmppsQueueResource.kt

```diff
@@ -12,7 +11,6 @@ import org.springframework.web.bind.annotation.RestController
 
 @RestController
 @RequestMapping("/queue-admin")
-@ConditionalOnExpression("'\${hmpps.sqs.reactiveApi:false}'.equals('false')")
```

Comment:
the constructor is called from `HmppsSqsConfiguration` so this conditional is not applied anyway and ignored